test/cli/help/main/expected-stdout

```diff
@@ -61,6 +61,7 @@ Available Commands:
   remote table import            Import data to a table.
   remote table preview           Preview a table's data.
   remote table unload            Export a table's data into file storage.
+  remote table upload            Upload file to Keboola Staging Storage.
```

Comment:
Tu je preklep ` Upload file to Keboola Staging Storage.`, kedze to subor nahraje do tabulky.

Reply:
Špatně jsem zadal ten path pro help msg v commandu (byl tam `file/upload` místo `table/upload`)